packages/components/src/components/tag/tag-shared-utils.ts

```diff
@@ -0,0 +1,45 @@
+import { getInsetJssStyle, ThemedColors } from '../../styles';
```

Comment:
I created this file in order to place here all the constants which are being used in "tag-dismissible", so that changing "tag-styles.ts" doesn't affect "tag-dismissible"'s chunk size